src/System.Security.Cryptography.Algorithms/tests/RijndaelTests.cs

```diff
@@ -52,6 +52,27 @@ private static void VerifyDefaults(Rijndael alg)
         }
 
         [Fact]
+        public static void VerifyBlocksizeIVNulling()
+        {
+            using (var testIVAlg = Rijndael.Create())
+            {
+                using (var alg = Rijndael.Create())
+                {
+                    alg.IV = testIVAlg.IV;
+                    alg.BlockSize = 128;
+                    Assert.Equal(testIVAlg.IV, alg.IV);
+                }
+
+                using (var alg = new RijndaelManaged())
+                {
+                    alg.IV = testIVAlg.IV;
+                    alg.BlockSize = 128;
+                    Assert.Equal(testIVAlg.IV, alg.IV);
+                }
+            }
```

Comment:
Probably good to make a custom Rijndael type which doesn't necessarily work, but extends the Rijndael class, so that if anyone happens to have done so we can check that it behave netfx-ishly.

Reply:
Good idea. Added.